openapi/index.yaml

```diff
@@ -132,4 +147,17 @@ definitions:
     $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v14.6.0/openapi/definitions.yaml#/ServiceTupleCollection"
   PaginationResponse:
     $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v14.6.0/openapi/definitions.yaml#/PaginationResponse"
+  ServiceId:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v14.6.0/openapi/definitions.yaml#/ServiceId"
+  ServicesByScope:
+    type: object
+    properties: 
+      NATIONAL:
+        type: array
+        items:
+          $ref: "#/ServiceId"
+      LOCAL:
+        type: array
+        items:
+          $ref: "#/ServiceId"
```

Comment:
Is empty object a valid `ServicesByScope`?

Reply:
I guess it's a valid object, because if we don't have any visible service we correctly return an empty object.